pipelines/rj_smas__disparo_pic_lembrete/validators.py

```
@@ -0,0 +1,207 @@
+        return payload
+
+    except ValidationError as e:
+        error_details = []
+        for error in e.errors():
+            field = error.get("loc", ["unknown"])[0]
+            message = error.get("msg", "Erro desconhecido")
+            error_details.append(f"{field}: {message}")
+
+        error_msg = f"Payload inválido: {'; '.join(error_details)}"
+        log(f"ERRO: {error_msg}")
+        raise ValidationError(error_msg)
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)